src/common/Routes.ts

```diff
@@ -8,7 +8,8 @@ export enum RoutePaths {
   LOGIN = '/login',
   DASHBOARD = "/dashboard",
   RECOMMENDATIONS = '/recommendations',
-  QUESTIONNAIRE = '/questionnaire',
+  QUESTIONNAIRE = '/ques/questionnaire',
```

Comment:
I don't think this is a good path name.
I would leave `/questionnaire` as the default path and show the questionnaire at that path.
`/questionnaire/results` would be the path for showing the results.

Can also do the same for Habit Tracker. The path `/habit-tracker` can be the Habits page
`/habit-tracker/analytics` can be the Analytics of the habit tracker.

It's a preference thing. But that would be my suggestion.